tests/JasmineHelper.ts

```diff
@@ -0,0 +1,51 @@
+import { SidetreeError } from '../lib/core/Error';
+
+/**
+ * Encapsulates the helper functions for the tests.
+ */
+export default class JasmineHelper {

```

Comment:
Not a fan of the name, perhaps:
```suggestion
export default class SidetreeErrorValidator {
```

Reply:
There's still Jasmine related code in here so I'm going to rename but keep Jasmine in the name.